eng/pipelines/azure-pipelines.yml

```diff
@@ -121,47 +91,41 @@ stages:
         steps:
           - template: /eng/pipelines/templates/BuildAndTest.yml
             parameters:
-              runAsPublic: ${{ eq(variables._RunAsPublic, True) }}
+              runAsPublic: true
```

Comment:
Is this file now used only for the internal pipeline? If yes, then should this be false?

Reply:
oh, this is what you want for the public one and are testing in this file :D

Reply:
Yes, this is `true` for the public pipelines only.